packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -29,9 +29,11 @@ function installFrontierDependencies(appPath, appName, ownPath) {
       '@fs/zion-subnav',
       '@fs/zion-user',
       '@fs/zion-ui',
+      'formik',
```

Comment:
lets pin these to the major versions like we do for the other 3rd party dependencies